pkg/cmd/pipeline/start_test.go

```diff
@@ -1869,12 +1888,8 @@ func Test_start_pipeline_last(t *testing.T) {
 	pipeline := Command(p)
 	got, _ := test.ExecuteCommand(pipeline, "start", pipelineName,
 		"--last",
-		"-r=git-repo=scaffold-git",
```

Comment:
The unit test for `tkn p start --last` wasn't really testing the feature since it overwrites many runtime values. The point of the test is to see whether the last pipelinerun's values are used, so I removed everything except `--last`.